installer-tests/run_tests.py

```diff
@@ -0,0 +1,155 @@
+import glob
+import os
+import pexpect
+import subprocess
+import re
+
+def _expect(line, current_cmd, do_re_escape=True, do_detect_slow=True, strip_comments=True,
+            verbose=True):
+    timeout = 1
+    if do_detect_slow:
+        if line.startswith('$[slow]'):
+            print 'Slow line...'
+            timeout = 30
+            line = line.replace('$[slow]', '', 1)
+
+    if verbose:
+        print 'expecting', line
+
+    if do_re_escape:
+        line = re.escape(line)
+
+    current_cmd.expect(line, timeout=timeout)
+
+
+TEST_ROOT=os.path.dirname(os.path.abspath(__file__))
+SANDSTORM_SOURCE_TREE=os.path.abspath(os.path.join(os.path.abspath(__file__), '..', '..'))
+SANDSTORM_IN_HOME=os.path.join(os.environ['HOME'], 'sandstorm')
+
+def vagrant_destroy():
+    subprocess.check_output(['vagrant', 'destroy', '-f'], cwd=TEST_ROOT)
+
+def handle_test_script(lines):
```

Comment:
If you plan to do more with this tooling, consider refactoring these into a TestScript class?  Something where the phases of (parse, read headers, interpret script) are more neatly structured?

I could see something like:

    class TestScript(object):
        def __init__(self, file_like_object):
            self.headers = []
            self.command_sequence = []
            self.preconditions = []
            self.postconditions = []
            self.parse(file_like_object) # does the stuff that iterating over lines currently does

        @staticmethod
        def load_from_file(path):
        ...
        def run(self):
            for condition in self.preconditions:
                self.verify_condition(condition)
            for command in self.command_sequence:
                self.run_command(command)
            for condition in self.post_conditions:
                self.verify_condition(condition)
        ....
        def verify_condition(self, condition):
             # dispatch on condition type
        ...

Or maybe I'm bikeshedding and this is totally fine for now. :P